jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    clearMocks: true,
```

Comment:
Will it be cleaner if jest config is written in package.json itself?

Reply:
Keeping in a separate file.